recipes/anuga/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "anuga" %}
+{% set version = "3.1.8" %}
+{% set sha = "1f15690caf02ed9c45d088536bee22d10a41d1954077e884b3cdcc361d422904" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/anuga-{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<3.6]
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('cxx') }}      # [not win]
+    - m2w64-gcc                  # [win]
+    - cython
+    - future
+    - setuptools
+    - wheel
+    - numpy
+    - dill
+    - libpython  # [win]
+  
+  host:
+    - python 
+    - cython
+    - libpython  # [win]
+    - dill
+    - future
+    - gdal
+    - gitpython
+    - matplotlib-base
+    - meshpy
+    - netcdf4
+    - numpy
+    - Pmw
+    - pymetis
+    - pytest
+    - pytz
+    - scipy
+    - setuptools
+    - utm
+    - wheel
+    - pip
+  run:
+    - python
+    - m2w64-gcc-libs  # [win]
+    - libpython       # [win]
+    - libstdcxx-ng    # [linux]
+    - libgcc-ng       # [linux]
```

Comment:
```suggestion
```
These are added automatically as run_exports from the compilers, and many times the cxx standard library isn't even used.

Reply:
@carterbox  removed as suggested.